_posts/2023-12-29-welcome-to-jekyll.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Welcome to Jekyll!"
+date:   2023-12-29 17:53:27 +0000
+categories: jekyll update
```

Comment:
🗣️ This publishes to the following path (with optional suffix of `.html` or `/`):

* `/jekyll/update/2023/12/29/welcome-to-jekyll`

Clearing `categories` moves `2023` up to the root:

* `/2023/12/29/welcome-to-jekyll`